README.md

```diff
@@ -67,6 +67,7 @@ config :sentry,
 | `release` | False  | None | |
 | `server_name` | False  | None | |
 | `use_error_logger` | False  | False | |
+| `hackney_opts` | False  | [] | |
```

Comment:
We should link to the hackney request opts https://github.com/benoitc/hackney/blob/master/doc/hackney.md#request5